models/token.js

```diff
@@ -9,12 +9,9 @@ const MODEL = {
         .integer()
         .positive(),
 
-    uuid: Joi
+    value: Joi
         .string()
```

Comment:
I don't see anything in iron talking about preservation of length or the contents of the sealed string.